### PR TITLE
Cucumber Scenario for Client

### PR DESCRIPTION
**Pull request by shahharsh06**

### CODE REVIEW

**Review by shweta-kumaran — COMMENTED** [review]
Looks good to me, I am unable to check if it passes if @rahulb99 can verify?

File: spec/features/ClientHomepage.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Feature: Client Profile page
+  As a client
+  I want to see a list of my projects
+  So that I can manage them
+
+  Scenario: Client can see list of projects
+    Given a client with email "client@example.com" and password "client123" is logged in as a "client"
+    When the client navigates to the Client Homepage
+    Then the client should see a list of projects
```
Comment (rahulb99) [inline spec/features/ClientHomepage.feature]: what is `projects` here?

File: spec/features/step_definitions/ client_homepage.steps.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            // Wait for the page to load
+            await driver.wait(until.elementLocated(By.id('projects')));
+        });
+
+        when('the client navigates to the Client Homepage', async () => {
+            // Navigate to the client homepage
+            await driver.get('http://127.0.0.1:3000/client');
+        });
+
+        then('the client should see a list of projects', async () => {
```
Comment (rahulb99) [inline spec/features/step_definitions/ client_homepage.steps.js]: same here

**Review by rahulb99 — CHANGES_REQUESTED** [review]

**Comment by rahulb99** [comment]
Also, merge the `main` branch and update README instructions to create a client with email and password, as mentioned in your feature file.

File: spec/features/ClientHomepage.feature (hunk 1 above)
Comment (shahharsh06) [reply spec/features/ClientHomepage.feature]: Correct in recent push. Please have a look at it.

File: spec/features/step_definitions/ client_homepage.steps.js (hunk 2 above)
Comment (shahharsh06) [reply spec/features/step_definitions/ client_homepage.steps.js]: Correct in recent push. Please have a look at it.

**Review by shweta-kumaran — APPROVED** [review]

**Review by rahulb99 — APPROVED** [review]
LGTM

**Pull request merged**